xamarin/realm-todo-dotnet/App.xaml.cs

```diff
@@ -44,24 +54,28 @@ protected override void OnStart()
             }
         }
 
-        private Realms.Sync.AppConfiguration LoadAppConfiguration()
+        private bool GetAppConfiguration()
```

Comment:
The naming here, at least, is questionable. "Get" implies it would return the app configuration. I think "HasAppConfiguration()" would be a better name for something returning a bool. However, why do we need this check? Since it's loading into member variables I think the best approach here would be leaving it as "LoadAppConfiguration", making it `void` and removing the try/catch block that just rethrows anyway.